libraries/Microsoft.Bot.PublishValidation/Progam.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Microsoft.Bot.PublishValidation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    class Program
+    {
+        // Return codes
+        private const int ERROR = 2;
+        private const int OK = 0;
+
+        public static int Main(string[] args)
+        {
+            var projectPath = args[0].ToString();
```

Comment:
This seems... really fragile. Is there a better way to do this?

Reply:
We used a PowerShell-like style to receive the parameters; this means, the boolean parameters are taken as true if present and the CSV parameters are taken with the values sent; and if not present, their default value is used